ninja/signature/details.py

```diff
@@ -159,6 +159,7 @@ def _create_models(self) -> TModels:
             )
 
             base_cls = param_cls._model
+            base_cls.model_config["from_attributes"] = True
```

Comment:
what's the logic for "from_attributes" here ? the signature module kinda works only on anayling function signatures...

Reply:
You are right, it was needed when I based my logic on `from_attributes` but its not valid anymore. Reverting